.github/workflows/reusable_test.yml

```diff
@@ -0,0 +1,191 @@
+name: Reusable build and test
+
+on: 
+  workflow_call:
+    inputs:
+      arch:
+        type: string
+        description: "Architecture to build the image for (amd64/arm64)"
+        required: true
+      build_scope:
+        type: string
+        description: "Mage build scope"
+        default: 'all'
+      build_target:
+        type: string
+        description: "Mage build target"
+        default: 'prod'
+      memgraph_version:
+        type: string
+        description: "Memgraph version built into this image (format: X.Y.Z)"
+        default: '2.18.0'
+      memgraph_download_link:
+        type: string
+        description: "Memgraph package download link. Leave empty to use the official download link."
+        required: false
+      memgraph_ref:
+        type: string
+        description: "Memgraph submodule branch to build query modules from"
+        default: 'master'
+      memgraph_ref_update:
+        type: string
+        description: "Update Memgraph submodule to the latest commit"
+        default: false
```

Comment:
I would put that to `true` since we want to test with the latest commit from branch

Reply:
done